routes/v1/user.js

```diff
@@ -87,9 +91,13 @@ const useUserRoutes = (useUser, mail) => {
   );
 
   const getAPIToken = prepauthToken(
-    {},
-    async (req, me) => {
-      const apiToken = await me.getAPIToken();
+    {
+      body: {
+        extraDynamicContent: { type: "/", optional: true },
+      },
```

Comment:
Neither this. It would be a security risk to allow the user to overwrite the JWT's content